pkg/3scale/amp/component/apicast.go

```diff
@@ -171,7 +171,8 @@ func (apicast *Apicast) StagingDeploymentConfig() *appsv1.DeploymentConfig {
 							},
 						},
 					},
-					PriorityClassName: apicast.Options.PriorityClassNameStaging,
+					PriorityClassName:         apicast.Options.PriorityClassNameStaging,
```

Comment:
what changed in this line?

Reply:
it's auto identation, code formatting done by my Goland IDE. It's because next line with Topology was added.
If it's critical, I can try find and disable the feature, but seems it's usefull.
(I tried restore the line, but after Save it's coming back :)

Reply:
I was curious. Golang fmt should take care of that.